ipaclient/install/ipa_certupdate.py

```diff
@@ -188,7 +188,8 @@ def update_server(certs):
         #
         logger.debug("resubmitting certmonger request '%s'", request_id)
         certmonger.resubmit_request(
-            request_id, ca='dogtag-ipa-ca-renew-agent-reuse', profile='')
+            request_id, ca='dogtag-ipa-ca-renew-agent-reuse',
+            profile='caCACert')
```

Comment:
[nit] Will it still work properly if we omit the `profile` kwarg?

Reply:
It looks like it will. Let me drop that. There is no need to override things here. Nice catch.